install/mysql/plugin_flyvemdm_empty.sql

```diff
@@ -259,3 +260,8 @@ CREATE TABLE `glpi_plugin_flyvemdm_taskstatuses` (
   INDEX `plugin_flyvemdm_tasks_id` (`plugin_flyvemdm_tasks_id`)
 ) ENGINE=MyISAM  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
 
+
+-- glpi_displaypreferences
+INSERT INTO `glpi_displaypreferences` (`id`, `itemtype`, `num`, `rank`, `users_id`)
+   VALUES (NULL,'PluginFlyvemdmFile', '1', '1', '0'),
+          (NULL,'PluginFlyvemdmFile', '4', '2', '0');
```

Comment:
Hi

This query will fail if for some reason the preference already exists, because of the unicity of (users_id, itemtype, num).

I think this is better to create  the row from PHP in the installation code.